app/models/child_item_request.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: child_item_requests
+#
+#  id              :bigint(8)        not null, primary key
+#  child_id        :bigint(8)
+#  item_request_id :bigint(8)
+#  created_at      :datetime         not null
+#  updated_at      :datetime         not null
+#
+
+class ChildItemRequest < ApplicationRecord
+  belongs_to :item_request
+  belongs_to :child
+
+  def quantity
+    item_request.quantity.to_i / item_request.children.size
```

Comment:
What happens if we have quantity == 100 and children.size == 3? Do we need to cast one of these operands to a float so that the quantity is 33.3333 instead of 33?

Reply:
I think we'll be refactoring that part anyways - I really just pushed this up to get a clear view of what it looked like once master was merged in